src/main/java/com/github/pires/obd/reader/activity/MainActivity.java

```diff
@@ -659,6 +664,18 @@ public void onGpsStatusChanged(int event) {
         }
     }
 
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        if (requestCode == REQUEST_ENABLE_BT) {
+            if (resultCode == Activity.RESULT_OK) {
+                btStatusTextView.setText(getString(R.string.status_bluetooth_connected));
+            } else {
+                Toast.makeText(this, "Bluetooth not enabled", Toast.LENGTH_LONG).show();
```

Comment:
Yeah that just skipped my mind, I ll make the necessary changes.
On 26-Dec-2015 4:43 pm, "Paulo Pires" <notifications@github.com> wrote:

> In src/main/java/com/github/pires/obd/reader/activity/MainActivity.java
> <https://github.com/pires/android-obd-reader/pull/162#discussion_r48446613>
> :
>
> > @@ -659,6 +664,18 @@ public void onGpsStatusChanged(int event) {
> >          }
> >      }
> >
> > +    @Override
> > +    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
> > +        if (requestCode == REQUEST_ENABLE_BT) {
> > +            if (resultCode == Activity.RESULT_OK) {
> > +                btStatusTextView.setText(getString(R.string.status_bluetooth_connected));
> > +            } else {
> > +                Toast.makeText(this, "Bluetooth not enabled", Toast.LENGTH_LONG).show();
>
> Why not reuse getString(R.string.text_bluetooth_disabled)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pires/android-obd-reader/pull/162/files#r48446613>.
>